docs/developing/troubleshooting.md

```diff
@@ -63,3 +63,27 @@ If your app wants to write to locations other than `/var`, and it's not easy to
 change the app's code, one work-around is to create a symlink from the location
 you app wants to modify to a location under `/var`.  This won't work for all
 applications however.
+
+## ETag data
+
+Sandstorm drops invalid ETag data. This typically has no impact on app functionality. When Sandstorm
```

Comment:
It's surprising that this section would open by talking about an error case. Because of this, especially when skimming, it's easy for people to miss the word "invalid" and think that this line is saying that Sandstorm drops all ETags.

Suggestion:

> Sandstorm normally allows valid `ETag` headers to pass through to the client. However, the HTTP standard requires `ETag` values to take a specific form -- for example, they must be quoted. Many applications incorrectly provide non-quoted values for this header. Sandstorm does not permit invalid header values because ambiguity can lead to security problems. Therefore, Sandstorm will drop invalid `ETag` headers and will write a warning to the grain's debug log. Usually, this does not affect app functionality, other than to reduce the effectiveness of the browser's cache.